Formula/vault-cli.rb

```diff
@@ -6,6 +6,11 @@ class VaultCli < Formula
   license "Apache-2.0"
   head "https://github.com/apache/jackrabbit-filevault.git"
 
+  livecheck do
+    url "https://repo1.maven.org/maven2/org/apache/jackrabbit/vault/vault-cli/"
+    regex(%r{href=.*?(\d+(?:\.\d+)+)/?["' >]}i)
```

Comment:
```suggestion
    url "https://search.maven.org/remotecontent?filepath=org/apache/jackrabbit/vault/vault-cli/"
    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
```

This is the standard regex we use for matching relative version directories on a directory listing page. The notable difference is the use of `["']?`, which matches the start of the `href` attribute to restrict it only to directories like `1.2.3/` and not something like `example-1.2.3`. This isn't much of a concern right now but we encounter unexpected things all the time.